### ISSUE TITLE
Crash report

### ISSUE THREAD

**Issue by saintech** [Tobi823/ffupdater#417]
Stacktrace:
```
de.marmaro.krt.ffupdater.network.exceptions.NetworkException: File was not downloaded: /storage/emulated/0/Android/data/de.marmaro.krt.ffupdater/files/Download/com_brave_browser_1_52_122.apk
	at de.marmaro.krt.ffupdater.installer.ApkChecker$Companion.throwIfDownloadedFileHasDifferentSize(ApkChecker.kt:29)
	at de.marmaro.krt.ffupdater.DownloadActivity.startDownload(DownloadActivity.kt:315)
	at de.marmaro.krt.ffupdater.DownloadActivity.access$startDownload(DownloadActivity.kt:61)
	at de.marmaro.krt.ffupdater.DownloadActivity$startDownload$1.invokeSuspend(DownloadActivity.kt:0)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at android.os.Handler.handleCallback(Handler.java:883)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loop(Looper.java:214)
	at android.app.ActivityThread.main(ActivityThread.java:7356)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:491)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:940)
```
Device information:
| Key | Value |
| --- | --- |
| FFUpdater version | 77.7.14 (140) release |
| Device | GS290 (GS290_EEA, GS290, k63v2_64_bsp) |
| Manufacturer | Gigaset (Gigaset) | 
| Supported ABIs | arm64-v8a, armeabi-v7a, armeabi |
| Android version | 10 (SDK: 29) |
| OS | runner-6a212837-project-53-concurrent-0, root, dev-keys,dev-release, 1608047995000 |         

The error appears after the APK is fully loaded. The download progress is as usual.

**Comment by crepp1967** [comment]
Stacktrace:
```
de.marmaro.krt.ffupdater.background.BackgroundException: The background job fails due to an unrecoverable exception.
	at de.marmaro.krt.ffupdater.BackgroundJob.handleDoWorkException(BackgroundJob.kt:97)
	at de.marmaro.krt.ffupdater.BackgroundJob.access$handleDoWorkException(BackgroundJob.kt:52)
	at de.marmaro.krt.ffupdater.BackgroundJob$doWork$2.invokeSuspend(BackgroundJob.kt:86)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:104)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:570)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:749)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:677)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:664)
Caused by: de.marmaro.krt.ffupdater.network.exceptions.NetworkException: Can't find latest update for TOR_BROWSER
	at de.marmaro.krt.ffupdater.app.impl.AppBase.findAppUpdateStatus(AppBase.kt:109)
	at de.marmaro.krt.ffupdater.app.impl.AppBase$findAppUpdateStatus$1.invokeSuspend(AppBase.kt:0)
	... 6 more
```
Device information:
| Key | Value |
| --- | --- |
| FFUpdater version | 77.7.14 (140) release |
| Device | SM-G781B (r8qxeea, r8q, kona) |
| Manufacturer | samsung (samsung) | 
| Supported ABIs | arm64-v8a, armeabi-v7a, armeabi |
| Android version | 13 (SDK: 33) |
| OS | 21DKG912, dpi, release-keys, 1681978354000 |

**Comment by Tobi823** [comment]
Fixed in 78.0.1. If not, please create a new issue.
